src/sprints/sprints.service.ts

```diff
@@ -570,6 +570,47 @@ export class SprintsService {
             responsesArray,
         );
 
+        // find voyageNumber from sprint id, and make sure it matches the voyage the
+        // team member is part of
+        let voyageNumber;
+        const voyageNumberFromSprintId: any =
```

Comment:
could we use a better type than `any` here? From the variable name it looks like an id but it looks like a `prismaQuery`